src/guide/data_types.md

```diff
@@ -31,11 +35,25 @@ let user = { name, age }
 print(user) // { name: "Klaus Teuber", age: 42 }
 ```
 
-## Mutable record properties
+A catch with punning is that in order to pun the field for a record with a single field, you need to use a comma after the field name, to avoid accidentally creating a block expression with a single nested expression.
```

Comment:
Does this really need to be in the guide? Feels weird to call this out.

Reply:
I agree it's weird I think its helpful though and worth having. If we added an example I think it would make it more approachable and clear.

Reply:
To me this should be an aside or a footnote. Just being in the guide makes it seem like something you need to know, but most people wouldn't encounter this.

Reply:
I also think it would be useful to at least mention. I can get rid of this section and just incorporate an example into the block above

Reply:
I agree

Reply:
Yeah on second thought I'll just remove it. Agree about it providing little value